clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesManifest.java

```diff
@@ -139,6 +140,33 @@ public String getCreationTimestamp() {
     return result;
   }
 
+  @JsonIgnore
+  public Double getReplicas() {
```

Comment:
Why a `Double`? Replicas can't be a decimal

Reply:
i was surprised by this as well however the type returned for that field is Double so i left it as is.